javascript/ql/src/semmle/javascript/dataflow/Nodes.qll

```diff
@@ -284,6 +284,9 @@ DataFlow::SourceNode globalObjectRef() {
   or
   // `require("global")`
   result = moduleImport("global")
+  or
+  // Closure library - based on AST to avoid recursion with Closure library model
+  result.asExpr().(DotExpr).getQualifiedName() = "goog.global"
```

Comment:
Do we perhaps want to use `globalVarRef` to ensure `goog` is a global and not a local of the same name?